SpringCollab2020Module.cs

```diff
@@ -14,9 +15,21 @@ public class SpringCollab2020Module : EverestModule {
         }
 
         public override void Load() {
+            Everest.Events.Level.OnExit += onLevelExit;
         }
 
         public override void Unload() {
-        }
+            Everest.Events.Level.OnExit -= onLevelExit;
```

Comment:
Hooks and events shouldn't need to be removed on unload. I've already got a solution partially ready in case mod unloading ever gets implemented, but the existing documentation (namely "remove any event listeners and undo all hooks") ~~is~~ was still outdated as it didn't really matter 😅 

Feel free to keep this line here though as it's a good practice no matter what.